src/components/homepage/Homepage.test.js

```diff
@@ -1,12 +1,53 @@
-// import React from "react";
+import React, { useStateMock } from "react";
```

Comment:
ah, I think I see what the problem with the test might be. this should be { useState as useStateMock }